client/src/components/Spinner/Spinner.component.jsx

```diff
@@ -1,15 +0,0 @@
-import React from 'react'
-import './Spinner.styles.scss'
-
-import { ReactComponent as PageSpinner } from '../../assets/PageSpinner.svg'
-import { ReactComponent as ComponentSpinner } from '../../assets/three-dots.svg'
```

Comment:
you can take the opportunity to eliminate the two SVGs in this PR

Reply:
deleted